src/App.tsx

```diff
@@ -4,32 +4,71 @@ import logo from "../public/FischLogo.svg";
 
 import styles from "./App.module.css";
 import { Card } from "./components/Card";
-import { useRef, useState } from "react";
+import { JSX, useEffect, useState } from "react";
+import { NewCard } from "./components/NewCard";
 
-function App() {
-  const [cardsAmount, setCardsAmount] = useState(1)
-  const [cards, setCards] = useState([{}]);
+interface AppProps {
+  basicCards: {
+    face: string;
+    back: string;
+  }[];
+}
+
+interface CardObject {
+  face: string;
+  back: string;
+}
+
+function App(props: AppProps) {
+  const importedCards = props.basicCards.map((card, index) => {
+    return { face: card.face, back: card.back, key: index };
+  });
+
+  const [cardsArray, setCardsArray] = useState(importedCards);
+
+  const [incrementalKey, setIncrementalKey] = useState(cardsArray.length);
+
+  useEffect(
```

Comment:
**nitpick:** This is not how it's done in react, you don't need it. Let's remove it